lib/modules/dosomething/dosomething_kudos/includes/Kudos.php

```diff
@@ -86,6 +93,16 @@ public static function find(array $filters = []) {
   }
 
   /**
+   * Get the approved reactions.
+   * @return array
+   */
+  public static function getApprovedReactions() {
+    $kudos = new static;
```

Comment:
Yeah I'm not 100% sure what the bestest of practices is, but I've made the property `protected static` and the getter function `public` in the past & not been too upset with myself.